Index: src/Global_Variables/HtmlStrings.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Global_Variables/HtmlStrings.java	(date 1558961581026)
+++ src/Global_Variables/HtmlStrings.java	(date 1558961581026)
@@ -287,7 +287,7 @@
             "    <h1 style=\"font-family: sans-serif;left: 2%;position: absolute;color: white\">Welcome ";
 
     String FriendListPageDefaultDisplayHeaderEnd = "</h1>\n" +
-            "    <form action=\"/search\" method=\"post\" style=\"font-family: sans-serif;right: 5%;top: 5%;position: absolute;\">\n" +
+            "    <form action=\"\" method=\"post\" style=\"font-family: sans-serif;right: 5%;top: 5%;position: absolute;\">\n" +
             "        <input type=\"search\" placeholder=\"Search by name, email\" name=\"searchValue\">\n" +
             "        <input type=\"submit\" value=\"Go\">\n" +
             "    </form>\n" +
@@ -307,23 +307,4 @@
             "</form>\n" +
             "</body>\n" +
             "</html>";
-
-    String SearchPageDisplayTitle = "<body>\n" +
-            "<h1 style=\"font-family: sans-serif;position: relative; left: 47%;\">Results</h1>";
-
-    String SearchPageDisplayResultStart = "<div style=\"background-color: black;position: relative;margin-top: 2%; height: 8%; width: 75%;\">\n" +
-            "    <h2 style=\"font-family: sans-serif;position: relative;color: white;top: 25%;left: 5%;\">";
-
-    String SearchPageDisplayResultEnd = "</h2>\n" +
-            "    <form action=\"\" method=\"post\" style=\"font-family: sans-serif;position: relative;left: 90%;bottom: 45%;\">\n" +
-            "        <input type=\"submit\" value=\"Add Friend\">\n" +
-            "    </form>\n" +
-            "</div>";
-
-    String SearchPageDisplayResultEndAlreadyFriends = "</h2>\n" +
-            "    <h4 style=\"font-family: sans-serif;position: relative;left: 87.5%;bottom: 45%;color: white;\">Already Friends</h4>\n" +
-            "</div>\n";
-
-    String SearchPageDisplayNoResults = "<h2 style=\"font-family: sans-serif;position: relative; left: 45%;top: 5%;color: red\">No results found</h2>";
-
 }
